composer.json

```diff
@@ -21,7 +21,7 @@
         "ext-openssl": "*",
         "phpseclib/phpseclib": "^2.0",
         "spomky-labs/php-aes-gcm": "^1.2",
-        "symfony/process": "^3.4|^4.0"
+        "symfony/process": "^3.4|^4.1|^5.0"
```

Comment:
bump also 4.0 to 4.1, where array form was introduced
for 3.x versions, it was available already since 3,3
in 5.0 it is only array